brat/test/golden/error/fanin-diff-types.brat.golden

```diff
@@ -0,0 +1,9 @@
+Error in test/golden/error/fanin-diff-types.brat@FC {start = Pos {line = 2, col = 5}, end = Pos {line = 2, col = 13}}:
```

Comment:
What's up with this `FC` stuff?

Reply:
That's just the location of the error that's highlighted by the message. We could easily just print the start as `fanin-diff-types.brat@2:5` -- I'll open a subsequent PR for that